src/main/java/org/takes/rq/RqLive.java

```diff
@@ -132,6 +104,28 @@ public InputStream body() {
     }
 
     /**
+     * Checks whether or not the next byte to read is a Line Feed.
+     * @param input The input stream to read
+     * @param baos Current read header
+     * @param position Header line number
+     * @throws IOException If the next byte is not a Line Feed as expected.
+     */
+    private static void checkLineFeed(final InputStream input,
+        final ByteArrayOutputStream baos, final Integer position)
+        throws IOException {
+        if (input.read() != '\n') {
+            throw new HttpException(
+                HttpURLConnection.HTTP_BAD_REQUEST,
+                String.format(
+                    "there is no LF after CR in header, line #%d: \"%s\"",
```

Comment:
@essobedo this is kind of http://www.yegor256.com/2015/12/08/temporal-coupling-between-method-calls.html. We need to invoke this method only after CR, otherwise it will print message that is not true. I just propose in this case, to put info about CR in Javadoc, this should be enough